### ISSUE TITLE
How to cancel retrieving push notifications?

### ISSUE THREAD

**Issue by miri-am** [raix/push#61]
Is there a unregister method?

**Comment by raix** [comment]
Good call, we should prop. add a `Push.unregister` but what would the expected behaviour be?

**Comment by miri-am** [comment]
I dont know in detail how it works, but I think
1) remove entries for current userId from _raix_push_app_tokens.
2) Call unregister from PushPlugin: https://github.com/phonegap-build/PushPlugin/search?utf8=%E2%9C%93&q=unregister

**Comment by raix** [comment]
Should all details be removed or just the token? I'm not 100% sure but calling unregister might invalidate the token, and if the feedback service from gcm/apn registers this they should call the server and invalidate the token - this causes the token to be removed in the tokens collection. (the feedback stuff is already up and running - but it might be nice to test in relation to unregister)

**Comment by kvnadesara** [comment]
@raix Suppose I am logged-in to 2 devices at that time I got the notification correctly. But if I logout from one device, even I got notification in that device. So how to prevent this scenario that I should get notification in those devices in which I am currently logged-in.
